fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -516,6 +516,14 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = true,
     )
 
+    /**
+     * Indicates if the user has completed successfully first translation.
+     */
+    var isFirstTranslationSuccessfully: Boolean by booleanPreference(
```

Comment:
I'd prefer the naming to be something like, `showFirstTimeTranslation` or `isFirstTimeTranslation` just to make it a little more clear the intention here and elsewhere.

Reply:
I think the default should also be `true` and we will toggle it to false once the first translation is complete when we work on integration.

Reply:
Thanks for the update!